### DISCUSSION TITLE
Urgent: Lost Domain Name Transfer Issue

### DISCUSSION THREAD

**Discussion by dji114** [vercel/community#4508] [Account Issue] [answered]
### Question

I am writing to seek urgent assistance with a domain name transfer issue that has arisen with my Vercel account. I am feeling quite distressed about this situation, and I would greatly appreciate your help in resolving it.

Here's the sequence of events that led to the problem:

I created an account with the email address for exemple '[thisisanexemple@gmail.com](mailto:thisisanexemple@gmail.com)' on Vercel.

Upon creating my account, two teams were generated: 'thisisanexemples-projects' and 'thisisanexemple-gmail.com.' The latter team ('thisisanexemple-gmail.com') does not appear anywhere in my account or in the Teams section, making it inaccessible.

Subsequently, I purchased a domain name for $20, and I intended to transfer it to my Vercel account.

I initiated a transfer from my account's Domains section, aiming to send the domain to the 'thisisanexemple-gmail.com' team, but, unfortunately, I couldn't find the team anywhere.

Regrettably, I lost track of the domain name as it now seems to have disappeared, neither showing in my account nor in the 'thisisanexemple-gmail.com' team that I initially intended to transfer it to.

In my attempts to locate the missing domain, I mistakenly paid another $20 for a new domain name, even before confirming the payment.

This situation has left me in a state of confusion and concern, as I am unable to locate the domain I initially purchased and am now faced with the financial repercussions of the accidental second purchase.

**Comment by amyegan** [comment, marked as answer]
Hi @dji114. I completely understand you not wanting to reveal identifiable information here in public. But I can't investigate without more info about your domain and account.

 If you have not done it already, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team and choose the `Billing` problem area. If you let me know your case number I can follow up with your ticket 🙇

**Reply by MasterchefEventDK** [reply]
I am writing here as I am facing the same transfer issue but not billing issue. How do I go about getting help with regarding the domain part of OP's question?

**Comment by dji114** [comment]
hello, thanks for your reply, i submitted the case.. but i counldn't find the 'case number' anywhere
maybe my acc ID will be helpful ?

**Comment by amyegan** [comment]
It looks like case 00166212 was solved recently, so I'm going to close this discussion. Please [start a new discussion](https://github.com/vercel/community/discussions/new) if you run into any other trouble 🙂
